init/91_webpasswd.sh

```diff
@@ -0,0 +1,3 @@
+if [ -z "$WEBPASSWD" ];then
```

Comment:
I am no bash hero, but according to the test's manpage, -z returns TRUE when the string being tested is empty, so shouldn't there be an exclamation mark ?

```
$ if [ -z "" ]; then echo "set password"; else echo "do nothing"; fi;
set password
```
```
$ if [ -z "p4ssw0rd" ]; then echo "set password"; else echo "do nothing"; fi;
do nothing
```